BlueBrain/Search
# Add support for parsing PubMed XMLs from command line

Fixes #433.

## How to test?
First, try the following snippet to see the available parameters for the parser.
```bash
$ bbs_database parse -h
usage: bbs_database parse [-h]
                          {CORD19ArticleParser,PubmedXMLParser} input_path
                          output_path

Parse article.
...
```
Second, try to parse a PMC XML file.
```bash
$ bbs_database parse PubmedXMLParser tests/data/sample_file.xml sample_file.pkl
```

## Checklist

  (if it is not the case, please create an issue first).
  (if needed)
  (if needed)
